--- RhinoPlugin.Test.Xunit/Tests/Test.OFCaseExecution.cs
+++ RhinoPlugin.Test.Xunit/Tests/Test.OFCaseExecution.cs
@@ -134,6 +134,8 @@
                 CreateNoWindow = true,
                 WindowStyle = ProcessWindowStyle.Hidden
             };
+            startInfo.ArgumentList.Add("/c");
+            startInfo.ArgumentList.Add(batchFilePath);

             using (var process = Process.Start(startInfo))
